Resources/Prototypes/_NF/Entities/Objects/Materials/scrap.yml

```
@@ -0,0 +1,62 @@
+#region base
+- type: entity
+  id: NFBaseScrap
+  parent: BaseScrap
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Materials/Scrap/generic.rsi
+  - type: Item
+    sprite: _NF/Objects/Materials/Scrap/generic.rsi
+
+#region variations
+- type: entity
+  id: NFScrapSilver
+  parent: NFBaseScrap
+  name: fried control panel
+  description: A control panel of sorts with fried circuitry. You could likely extract some materials out of this.
+  components:
+  - type: Sprite
+    layers:
+    - state: junk-apc-1
+      map: [ "base" ]
+  - type: Item
+    heldPrefix: "panel"
+    size: Large
+    shape:
+    - 0,0,2,2 # 3x3
+  - type: RandomSprite
+    available:
+    - base:
+        junk-airalarm-1: ""
+        junk-apc-1: ""
+        junk-panel-1: ""
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 500 # 5 sheets
+      RawScrap: 1000 # 10 sheets
+      RawScrapTech: 200 # 2 sheets
+      Silver: 500 # 5 sheets
+
+- type: entity
+  id: NFScrapSolarGlass
+  parent: NFBaseScrap
+  name: ruined solar panel
+  description: An utterly ruined solar panel. You could likely extract some materials out of this.
+  components:
+  - type: MultiHandedItem
+  - type: Sprite
+    layers:
+    - state: junk-solar-1
+      map: [ "base" ]
+  - type: Item
+    heldPrefix: "solar"
+    size: Large
```

```suggestion
    size: Huge
```
At 5x5, this thing's size would generally be huge.  Is there a reason it's only large?